src/main/java/org/opensearch/security/dlic/rest/validation/AbstractConfigurationValidator.java

```diff
@@ -297,4 +316,25 @@ public String getMessage() {
     protected final boolean hasParams() {
         return param != null && param.length > 0;
     }
+
+    private boolean hasNullElement(JsonNode currentNode) {
+        if (currentNode.isArray()) {
+            List contentArray = DefaultObjectMapper.objectMapper.convertValue(currentNode, java.util.List.class);
```

Comment:
Is this conversion necessary? If it is an array, can you cast it to `ArrayNode`?

Reply:
Used `ArrayNode` in the latest commit